pkg/networkutils/network.go

```diff
@@ -301,13 +309,20 @@ func (n *linuxNetwork) SetupHostNetwork(vpcCIDR *net.IPNet, vpcCIDRs []*string,
 	type snatCIDR struct {
 		cidr        string
 		isExclusion bool
+		isInclusion bool
 	}
 	var allCIDRs []snatCIDR
 	for _, cidr := range vpcCIDRs {
-		allCIDRs = append(allCIDRs, snatCIDR{cidr: *cidr, isExclusion: false})
+		allCIDRs = append(allCIDRs, snatCIDR{cidr: *cidr, isExclusion: false, isInclusion: false})
```

Comment:
so what is neither an inclusion nor an exclusion ?

Reply:
vpcCirds wasn't an exclusion for them at the beginning... It's juste for commenting what's come from "include" or "exclude" or neither parameters. (vpc cidrs are automaticly added)